cmd/boulder-ra/main.go

```diff
@@ -78,13 +78,14 @@ type Config struct {
 		// limits are per section 7.1 of our combined CP/CPS, under "DV-SSL
 		// Subscriber Certificate". The value must match the CA and WFE
 		// configurations.
-		MaxNames int `validate:"required,min=1,max=100"`
+		// Deprecated: Set ValidationProfiles[*].MaxNames instead.
```

Comment:
```suggestion
        //
		// Deprecated: Set ValidationProfiles[*].MaxNames instead.
```

Reply:
Optional: We seem to have both styles in our codebase, I personally prefer the extra line.